compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -2010,6 +2010,7 @@ pub struct FmtPrinterData<'a, 'tcx> {
 
     pub ty_infer_name_resolver: Option<Box<dyn Fn(ty::TyVid) -> Option<Symbol> + 'a>>,
     pub const_infer_name_resolver: Option<Box<dyn Fn(ty::ConstVid) -> Option<Symbol> + 'a>>,
+    pub for_suggestion: bool,
```

Comment:
in general, it's probably best for code maintainability to document when you add new fields like this that don't have obvious side-effects. especially since "for_suggestion" doesn't really indicate what it does to a pretty printer.

Reply:
but also like... why is this called `for_suggestion`? this could be named something like `print_qualified_impl` or something, i think. you can probably come up with a better name for this.